config/routes.rb

```diff
@@ -10,6 +10,7 @@
   post   '/login',   to: 'sessions#create'
   delete '/logout',  to: 'sessions#destroy'
   resources :users
+  resources :guides
```

Comment:
```suggestion
  resources :guides, only: %i[new create]
```
We should only enable the routes we are supporting.